React/CoreModules/RCTDeviceInfo.mm

```diff
@@ -192,15 +195,15 @@ - (void)_interfaceOrientationDidChange
   // Update when we go from portrait to landscape, or landscape to portrait
   // Also update when the fullscreen state changes (multitasking) and only when the app is in active state.
   if ((isOrientationChanging || isResizingOrChangingToFullscreen) && isActive) {
```

Comment:
isActive is used only once, here. You can use the `RCTIsAppActive()` function directly instead of saving its value in a variable.